capstone/src/components/User/Adventurer/FindAdventure/Map/Map.jsx

```diff
@@ -1,35 +1,40 @@
 import './Map.css';
-import { useState, useRef, useEffect } from 'react';
+import { useState, useEffect, useContext } from 'react';
+import axios from 'axios';
 import { useLoadScript, GoogleMap, MarkerF } from '@react-google-maps/api';
+import AdventurerContext from '../../../../../Contexts/AdventurerContext';
```

Comment:
nit: If this already works fine for you, just skip over this. But relative imports can sometimes get annoying. 

If you want to simplify imports to be like `import X from "src/my/module/here.js"`, instead of `import X from "../../../../../module/here.js"`... you could look here: https://create-react-app.dev/docs/importing-a-component/#absolute-imports